drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -6,7 +6,7 @@
 menuconfig ETH_SAM_GMAC
 	bool "Atmel SAM Ethernet driver"
 	depends on SOC_FAMILY_SAM
-	select NOCACHE_MEMORY
+	select NOCACHE_MEMORY if !SOC_SERIES_SAM4E
```

Comment:
instead of using SAM4E, I would advise to use:
select NOCACHE_MEMORY if ARCH_HAS_NOCACHE_MEMORY_SUPPORT

Reply:
Done!